mesonbuild/coredata.py

```diff
@@ -509,13 +587,18 @@ def set_default_options(self, default_options, subproject, env):
         # Some options default to environment variables if they are
         # unset, set those now. These will either be overwritten
         # below, or they won't.
-        options['pkg_config_path'] = os.environ.get('PKG_CONFIG_PATH', '').split(':')
+        if not 'pkg_config_path' in env.cmd_line_options:
```

Comment:
__[Flake8]__ [E713] test for membership should be 'not in' ([link](https://sider.review/gh/repos/19784232/pulls/5263#issue-3981248))

<sub>posted by Sider</sub>